tests/system/test_table_admin.py

```diff
@@ -354,3 +350,103 @@ def test_table_backup(
     restored_table = alt_instance.table(restored_table_id)
     assert restored_table in alt_instance.list_tables()
     restored_table.delete()
+
+
+def test_table_copy_backup_to_new_instance(
+    admin_client,
+    unique_suffix,
+    instance_labels,
+    location_id,
+    data_instance_populated,
+    data_cluster_id,
+    instances_to_delete,
+    tables_to_delete,
+    backups_to_delete,
+    skip_on_emulator,
+):
+    from google.cloud._helpers import _datetime_to_pb_timestamp
+    from google.api_core.exceptions import FailedPrecondition
+    from google.cloud.bigtable.backup import Backup
+
+    source_instance = data_instance_populated
+    temp_table_id = "test-backup-table"
+    source_table = source_instance.table(temp_table_id)
+    source_table.create()
+    tables_to_delete.append(source_table)
+
+    temp_backup_id = "test-backup"
+
+    expire = datetime.datetime.now(tz=datetime.timezone.utc) + datetime.timedelta(
+        days=7
+    )
+
+    # create a backup to be copied
+    temp_backup = source_table.backup(
+        temp_backup_id,
+        cluster_id=data_cluster_id,
+        expire_time=expire,
+    )
+    temp_backup.create()
+    backups_to_delete.append(temp_backup)
+
+    # Test copying to the same instance
+    backup_op = temp_backup.copy("copied-backup")
```

Comment:
let's add a unique identifier to `copied-backup`

Reply:
I've added `-{unique_suffix}` to the copy backup ids